plugin.json

```diff
@@ -22,42 +22,42 @@
         "settings": [
             {
                 "key": "JitsiURL",
-                "display_name": "Jitsi Server URL",
+                "display_name": "Jitsi URL",
                 "type": "text",
                 "help_text": "The URL for an on-premise Jitsi server. For example, https://jitsi.example.com.",
                 "placeholder": "https://jitsi.example.com"
             },
             {
                 "key": "JitsiJWT",
-                "display_name": "Enable Jitsi JWT",
+                "display_name": "Use JWT Authentication for Jitsi",
                 "type": "bool",
-                "help_text": "Enable this, if You have a Jitsi with JWT"
+                "help_text": "(Optional) If your Jitsi server uses JSON Web Tokens (JWT) for authentication, set this value to true."
             },
             {
                 "key": "JitsiAppID",
-                "display_name": "Jitsi Server app ID",
+                "display_name": "App ID for JWT Authentication",
                 "type": "text",
-                "help_text": "The App ID on Your Jitsi JWT server"
+                "help_text": "(Optional) The App ID used for authentication by the Jitsi server and JWT token generator."
             },
             {
                 "key": "JitsiAppSecret",
-                "display_name": "Jitsi Server app secret",
+                "display_name": "App Secret for JWT Authentication",
                 "type": "text",
-                "help_text": "The App secret on Your Jitsi JWT server"
+                "help_text": "(Optional) The App Secret used for authentication by the Jitsi server and JWT token generator."
             },
             {
                 "key": "JitsiLinkValidTime",
-                "display_name": "Link valid time, in minutes",
+                "display_name": "Meeting Link Expiry Time (minutes)",
                 "type": "number",
-                "help_text": "How many minutes is the generated link active, if set smaller then 1, valid time is automaticaly set to 30 minutes.",
+                "help_text": "(Optional) The number of minutes from when the meeting link is created to when it becomes invalid. Minimum is 1 minute.",
                 "default": 30
             },
             {
                 "key": "JitsiNamingScheme",
-                "display_name": "Meeting naming scheme",
+                "display_name": "Meeting Naming Scheme",
                 "type": "radio",
                 "help_text": "How should the plugin choose a meeting name? Some names may cause conflicts if you use them on a public server, like meet.jit.si.",
-                "default": "english-titlecase",
+                "default": "uuid",
```

Comment:
I am proposing `uuid` as the default option, but open to feedback. This is consistent with other video plugins for Mattermost and ensures uniqueness.

Reply:
Jitsi Meet uses english titlecase meeting IDs by default, so this might be jarring for people who are used to Jitsi's names

Reply:
> Jitsi Meet uses english titlecase meeting IDs by default, so this might be jarring for people who are used to Jitsi's names

Ah, this is really useful, thank you for sharing. Then the english titlecase meeting IDs make sense for a consistent experience for Jitsi users.

Reply:
I think I agree with @lediur the Jitsi default behavior is expected here.

Reply:
Thank you @lediur and @jespino! I will revert back to `english-titlecase` since it's default behaviour in Jitsi.